### PR TITLE
New Windows install include

### PR DESCRIPTION
**Pull request by maguinis**
_Description of what this PR is changing or adding, and why:_

_Issues fixed by this PR (if any):_ Fixes #ISSUE-NUMBER

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/get-started/install/_get-sdk-win.md
Comment (kevmoo) [inline src/get-started/install/_get-sdk-win.md:54]:
```suggestion
{% include docs/dart-tool-win.md %}
```

File: src/get-started/install/_get-sdk-win_v2.md
Comment (johnpryan) [inline src/get-started/install/_get-sdk-win_v2.md:7]: Should we use a URL like "windows-quick" instead?

```suggestion
  path: /get-started/windows-quick
```

File: src/get-started/install/windows_quick.md
Comment (johnpryan) [inline src/get-started/install/windows_quick.md:7]: Will this conflict with the existing [get-started/editor](https://docs.flutter.dev/get-started/editor) page?

File: src/get-started/install/_get-sdk-win_v2.md
Comment (domesticmouse) [inline src/get-started/install/_get-sdk-win_v2.md:91]: We should consider changing the language here and below, as missing dependencies for things the user is not intending to use is A-OK. For example, not having desktop compilers for developers wanting to do Android development, or missing Android Studio plugins when the developer is going to use VS Code.

**Comment by domesticmouse** [comment]
Also, can we give this PR a more descriptive title? I'm guessing something along the lines of "New Windows install include" or similar?

File: src/get-started/install/_get-sdk-win_v2.md
Comment (sfshaza2) [inline src/get-started/install/_get-sdk-win_v2.md:43]:
```suggestion
You are now ready to run Flutter commands in the Flutter console.
```

File: src/get-started/install/windows_quick.md
Comment (sfshaza2) [reply src/get-started/install/windows_quick.md:7]: Good question. This PR needs to be staged to be sure.

**Review by sfshaza2 — COMMENTED** [review]
A few minor comments, but I agree with @johnpryan that we should double check this when staged.

**Comment by johnpryan** [comment]
Staged here: https://flutter-website-jr-stagi-988d4.web.app

File: src/get-started/install/windows_quick.md
Comment (johnpryan) [reply src/get-started/install/windows_quick.md:7]: I misread this line, this is the `next` URL in the getting started flow.

File: src/get-started/install/windows_quick.md
Comment (johnpryan) [inline src/get-started/install/windows_quick.md:30]: Make sure you leave out any of these `% include`s you don't want, right now the full Android instructions are being included on this page.

```suggestion
```

File: src/get-started/install/_get-sdk-win_v2.md
Comment (loic-sharma) [inline src/get-started/install/_get-sdk-win_v2.md:31]:
```suggestion
  `C:\Program Files\` that require elevated privileges.
```

File: src/get-started/install/_get-sdk-win_v2.md
Comment (loic-sharma) [reply src/get-started/install/_get-sdk-win_v2.md:43]: What's the Flutter console? Consider using the same phrase as the macOS guide:

```suggestion
You are now ready to run Flutter commands.
```

File: src/get-started/install/_get-sdk-win_v2.md
Comment (loic-sharma) [inline src/get-started/install/_get-sdk-win_v2.md:62]: Nit: you don't necessarily need to close the terminal for these changes to take effect, consider simplifying to:

```suggestion
You will need to launch a new console window
for these changes to take effect.
```

File: src/get-started/install/_get-sdk-win_v2.md
Comment (loic-sharma) [inline src/get-started/install/_get-sdk-win_v2.md:98]:
```suggestion
  or Dart plugin of Android Studio are not installed, move
```

File: src/get-started/install/windows_quick.md
Comment (loic-sharma) [inline src/get-started/install/windows_quick.md:41]: FYI `docs.microsoft.com` is now `learn.microsoft.com`, and, providing the `en-us` language code forces the reader to use `en-us` even if that is not their preferred language.

```suggestion
[Windows PowerShell 5.0]: https://learn.microsoft.com/powershell/scripting/install/installing-windows-powershell
```

File: src/get-started/install/_get-sdk-win_v2.md
Comment (atsansone) [inline src/get-started/install/_get-sdk-win_v2.md:32]: **issue:** This doesn't need to be two blocks.
```suggestion
{{site.alert.warning}}
  You cannot install Flutter to a path that contains special
  characters or spaces or requires elevated privileges.
  An example of both would be `C:\Program Files\`.
{{site.alert.end}}
```

File: src/get-started/install/_get-sdk-win_v2.md
Comment (atsansone) [reply src/get-started/install/_get-sdk-win_v2.md:56]: @loic-sharma : It was Windows 10.

File: src/get-started/install/_get-sdk-win_v2.md
Comment (atsansone) [inline src/get-started/install/_get-sdk-win_v2.md:53]: **issues:** 

1. [Style guide](https://developers.google.com/style/ui-elements#term-windows) uses `In the...` rather than `From the...`.
2. This is an ordered list, so it should be numbered.
3. This first step is two steps. Separate them.

```suggestion
1. In the Start search bar, enter `env`.
2. Select **Edit environment variables for your account**.
```

File: src/get-started/install/_get-sdk-win_v2.md
Comment (atsansone) [inline src/get-started/install/_get-sdk-win_v2.md:56]: **issue:** The UI changed. Use current UI.
```suggestion
3. In the **Edit environment variables** dialog, look for an entry with `flutter\bin`.
```

File: src/get-started/install/_get-sdk-win_v2.md
Comment (atsansone) [inline src/get-started/install/_get-sdk-win_v2.md:59]:
```suggestion
  a. If the entry doesn't exist, click **New**.
  b.  Create a new entry with the full path to `flutter\bin` as its value.
  c. Click **Save**.
```

File: src/get-started/install/_get-sdk-win_v2.md
Comment (atsansone) [inline src/get-started/install/_get-sdk-win_v2.md:70]:
```suggestion
1. Open a console window.
2. Run the following command to see check for missing platform dependencies:
```

File: src/get-started/install/_get-sdk-win_v2.md
Comment (atsansone) [inline src/get-started/install/_get-sdk-win_v2.md:8]: **issue:** This text is displayed in the [staged version](https://flutter-website-jr-stagi-988d4.web.app/get-started/install/windows_quick).

File: src/get-started/install/_get-sdk-win_v2.md
Comment (atsansone) [inline src/get-started/install/_get-sdk-win_v2.md:12]: **suggestion:** If there are two ways to do this, consider making this a tabbed interface.

**Review by atsansone — CHANGES_REQUESTED** [review]
Some language and possible UI tweaks.

File: src/get-started/install/_get-sdk-win_v2.md
Comment (atsansone) [reply src/get-started/install/_get-sdk-win_v2.md:8]: Still an issue.

image

**Comment by sfshaza2** [comment]
It looks to me that this very involved PR is well in hand with @atsansone and @domesticmouse while I was out sick. Thanks, gang! I leave it to you to land when ready.

**Comment by ditman** [comment]
(I've created [an issue](https://github.com/flutter/website/issues/7741) for the failing `check_link_references` check)

**Comment by sfshaza2** [comment]
We are freezing the website today, by end of day. As this PR isn't complete, I can't land it. Please feel free to reopen after Flutter Forward to complete it. cc @atsansone

**Pull request closed without merging**
